### PR TITLE
ETH gas reporter

### PR DESCRIPTION
**Pull request by haythemsellami**
Use [eth-gas-reporter](https://www.npmjs.com/package/eth-gas-reporter) to get an idea about gas usage for the apps methods.

### CODE REVIEW

File: apps/coinsence-kit/contracts/CoinsenceKit.sol [hunk 1]
```diff
@@ -8,6 +8,7 @@ import "@aragon/os/contracts/lib/ens/PublicResolver.sol";
 import "./misc/APMNamehashOpen.sol";
 import "@coinsence/apps-space/contracts/Space.sol";
 import "@coinsence/apps-coin/contracts/Coin.sol";
+import "@aragon/os/contracts/apm/APMNamehash.sol";
```
Comment (bumi) [inline apps/coinsence-kit/contracts/CoinsenceKit.sol]: shoudl this go here?

**Review by bumi — COMMENTED** [review]
what is the reason that we do not enable that by default on `npm run test`?
I think that's a super valuable information?

File: apps/coinsence-kit/contracts/CoinsenceKit.sol (hunk 1 above)
Comment (haythemsellami) [reply apps/coinsence-kit/contracts/CoinsenceKit.sol]: ahh sorry will remove it... I was testing locally so I needed that.

**Comment by haythemsellami** [comment]
> what is the reason that we do not enable that by default on `npm run test`?
> I think that's a super valuable information?

Yup true... I changed that.

**Pull request merged**